lib/takeaway.rb

```diff
@@ -0,0 +1,38 @@
+require 'twilio-ruby'
+require 'sms'
+
+class Takeaway
+
+  attr_reader :menu, :confirm_order, :receipt, :total
+
+  def initialize
+    @menu = {"Burger"=>10, "Shake"=>5, "Fries"=>5, "Hot Dog"=>4, "Root Beer Float"=>0}  
```

Comment:
Metrics/LineLength: Line is too long. [88/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)